spanner/tagging_samples.rb

```diff
@@ -47,20 +47,24 @@ def transaction_tagging project_id:, instance_id:, database_id:
   client = spanner.client instance_id, database_id
 
   client.transaction request_options: { tag: "app=cart,env=dev" } do |tx|
-    venues = tx.execute \
-      "SELECT VenueId, VenueName, Capacity FROM Venues WHERE OutdoorVenue = @outdoor_venue",
-      params: { outdoor_venue: true },
-      request_options: { tag: "app=concert,env=dev,action=select" }
-
-    venues.rows.each do |venue|
-      capacity = (venue["Capacity"] / 4).round
-      tx.execute_update \
-        "UPDATE Venues SET Capacity = @capacity WHERE VenueId = @venue_id",
-        params: { venue_id: venue["VenueId"], capacity: capacity },
-        request_options: { tag: "app=concert,env=dev,action=update" }
-
-      puts "Capacity of #{venue['VenueName']} updated to #{capacity}"
-    end
+    tx.execute_update \
+      "UPDATE Venues SET Capacity = CAST(Capacity/4 AS INT64) WHERE OutdoorVenue = false",
+      request_options: { tag: "app=concert,env=dev,action=update" }
+
+    puts "Venue capacities updated."
+
+    tx.execute_update \
+      "INSERT INTO Venues (VenueId, VenueName, Capacity, OutdoorVenue) " \
+      + "VALUES (@venue_id, @venue_name, @capacity, @outdoor_venue)",
```

Comment:
This statement errors out because `LastUpdateTime` is a `NOT NULL` column.
Please use this instead:
```
"INSERT INTO Venues (VenueId, VenueName, Capacity, OutdoorVenue, LastUpdateTime) " \
+  "VALUES (@venueId, @venueName, @capacity, @outdoorVenue, PENDING_COMMIT_TIMESTAMP())"
```

Reply:
Actually, there is no `LastUpdateTime` in the table. Also, I took a look at the PR https://github.com/googleapis/python-spanner/pull/605/files, which does not have `LastUpdateTime` as well.